File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/fragment/BasicCustomUiFragment.java
package com.regula.facesample.items.customization.fragment;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;

import com.regula.facesample.R;
import com.regula.facesdk.fragment.FaceUiFragment;

/**
 * Created by Sergey Yakimchik on 19.08.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class BasicCustomUiFragment extends FaceUiFragment {

    @Override
    public View getSwapCameraButton(@NonNull View view) {
        return null;
    }

    @Override
    public View getFlashLightButton(@NonNull View view) {
        return null;
    }

    @Override
    public TextView getNotificationView(@NonNull View view) {
        return view.findViewById(R.id.notificationView);
    }

    @Override
    public int getResourceLayoutId() {
        return R.layout.custom_hint_view_appearance;
    }

    @Override
    public View getCloseButton(View view) {
        return view.findViewById(R.id.closeButton);
    }
}

File: BasicSample/app/src/main/java/com/regula/facesamplekotlin/items/fragments/BasicCustomUiFragment.kt
package com.regula.facesamplekotlin.items.fragments

import android.view.View
import android.widget.TextView
import com.regula.facesamplekotlin.R
import com.regula.facesdk.fragment.FaceUiFragment

class BasicCustomUiFragment: FaceUiFragment() {

    override fun getSwapCameraButton(view: View): View? {
        return null
    }

    override fun getFlashLightButton(view: View): View? {
        return null
    }

    override fun getNotificationView(view: View): TextView? {
        return view.findViewById(R.id.notificationView)
    }

    override fun getResourceLayoutId(): Int {
        return R.layout.custom_hint_view_appearance
    }

    override fun getCloseButton(view: View): View? {
        return view.findViewById(R.id.closeButton)
    }
}